server/controllers/constraints/need/create.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
+  pickup_at: {
+    numericality: {
+      greaterThan: Date.now()
+    }
+  },
+  pickup_latitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 90,
+    }
+  },
+  pickup_longitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 180,
+    }
+  },
+  dropoff_latitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 90,
+    }
+  },
+  dropoff_longitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 180,
+    }
+  },
+  requester_name: {
+    length: {
+      minimum: 3,
+    }
+  },
+  requester_phone_number: {
+    length: {
+      minimum: 8,
+    }
+  },
+  cargo_type: {
+    presence: true,
+    type: 'number',
```

Comment:
will it work with a numeric string?
Same question for next ones. All numeric values may come as strings

Reply:
It doesn't work for numeric strings. Not sure how to solve that.

Reply:
It does work for numeric strings if I used the `numericality` validation instead of type.